FindMe/Parse Manager/DatabaseManager.h

```diff
@@ -0,0 +1,22 @@
+//
+//  DatabaseManager.h
+//  FindMe
+//
+//  Created by Nicholas Wayoe on 7/21/20.
+//  Copyright © 2020 Nicholas Wayoe. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "Parse/Parse.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface DatabaseManager : NSObject
+
++ (void)createUser:(NSDictionary *)userDetails withCompletion:(void(^)(NSError *error))completion;
```

Comment:
This should probably take a `User` and save a `ParseUser`. Or something to that effect.